solr/contrib/ltr/src/java/org/apache/solr/ltr/rest/ManagedModelStore.java

```diff
@@ -177,7 +177,11 @@ public LTRScoringAlgorithm makeLTRScoringAlgorithm(Map<String,Object> map)
     if (map.containsKey(CommonLTRParams.MODEL_PARAMS)) {
```

Comment:
Seems like this entire "createParams(map)" if chunk is the same between the model + feature store.  Make a reusable function for both to use?